crates/storage_models/src/payment_intent.rs

```diff
@@ -122,6 +124,7 @@ pub enum PaymentIntentUpdate {
         business_label: Option<String>,
         order_details: Option<Vec<pii::SecretSerdeValue>>,
         metadata: Option<pii::SecretSerdeValue>,
+        udf: Option<pii::SecretSerdeValue>,
```

Comment:
we could remove this if udf is not intented to be updated...

Reply:
It is being updated in payment update and confirm call